### PR TITLE
Feature user

### PR DESCRIPTION
**Pull request by seamile**

### CODE REVIEW

File: common/middleware.py [hunk 1]
```diff
@@ -0,0 +1,25 @@
+from django.utils.deprecation import MiddlewareMixin
+
+from libs.http import render_json
+from common import stat
+
+
+class AuthMiddleware(MiddlewareMixin):
+    '''登录验证中间件'''
```
Comment (YangRong-brave) [inline common/middleware.py:8]: good job

**Comment by Super-H1** [comment]
通过

File: libs/http.py [hunk 2]
```diff
@@ -0,0 +1,21 @@
+import json
+
+from django.http import HttpResponse
+from django.conf import settings
+
+
+def render_json(data=None, code=0):
```
Comment (YangRong-brave) [inline libs/http.py:7]: 学到了，哈哈哈哈哈哈

**Comment by DCheng-zi** [comment]
I think you can be better

**Pull request merged**